FluentUI.Demo/src/main/java/com/microsoft/fluentuidemo/demos/V2PersonaListActivity.kt

```diff
@@ -236,6 +238,7 @@ class V2PersonaListActivity : DemoActivity() {
         val coroutineScope: CoroutineScope = rememberCoroutineScope()
         Scaffold(scaffoldState = scaffoldState) {
```

Comment:
Remove the Scaffold as this affect the internal element value. Use Toast to show onClick action

Reply:
Removed the scaffold usages in other PR